src/api/procedures/createGroup.ts

```diff
@@ -85,12 +85,16 @@ export async function prepareCreateGroup(
   const currentGroupPermissions = await P.map(groups, group => group.getPermissions());
 
   if (
-    currentGroupPermissions.some(({ transactions: transactionPermissions }) =>
-      isEqual(
-        optionize(orderTransactionPermissionsValues)(transactionPermissions),
-        optionize(orderTransactionPermissionsValues)(transactions)
-      )
-    )
+    currentGroupPermissions.some(({ transactions: transactionPermissions }) => {
+      if (transactionPermissions) {
```

Comment:
Only the inputs should be sorted, since this comes already sorted from the chain